analysis/topeft_run2/analysis_processor.py

```diff
@@ -582,29 +583,43 @@ def process(self, events):
 
                 # Trigger SFs
                 GetTriggerSF(year,events,l0,l1)
-                weights_obj_base_for_kinematic_syst.add(f"triggerSF_{year}", events.trigger_sf, copy.deepcopy(events.trigger_sfUp), copy.deepcopy(events.trigger_sfDown))            # In principle does not have to be in the lep cat loop
+                weights_obj_base_for_kinematic_syst.add(f"{year}", events.trigger_sf, copy.deepcopy(events.trigger_sfUp), copy.deepcopy(events.trigger_sfDown))            # In principle does not have to be in the lep cat loop
 
 
             ######### Event weights that do depend on the lep cat ###########
             select_cat_dict = None
-            with open(topeft_path("channels/ch_lst_test.json"), "r") as ch_json_test:
+
+            with open(topeft_path("channels/ch_lst.json"), "r") as ch_json_test:
                 select_cat_dict = json.load(ch_json_test)
 
-            # This dictionary keeps track of which selections go with which SR categories
-            if self.offZ_3l_split:
-                import_sr_cat_dict = select_cat_dict["OFFZ_SPLIT_CH_LST_SR"]
-            elif self.tau_h_analysis:
-                import_sr_cat_dict = select_cat_dict["TAU_CH_LST_SR"]
-            elif self.fwd_analysis:
-                import_sr_cat_dict = select_cat_dict["FWD_CH_LST_SR"]
-            else:
-                import_sr_cat_dict = select_cat_dict["TOP22_006_CH_LST_SR"]
+            if not self._skip_signal_regions:
+                # If we are not skipping the signal regions, we will import the SR categories
+                # This dictionary keeps track of which selections go with which SR categories
+                if self.offZ_3l_split:
+                    import_sr_cat_dict = select_cat_dict["OFFZ_SPLIT_CH_LST_SR"]
+                elif self.tau_h_analysis:
+                    import_sr_cat_dict = select_cat_dict["TAU_CH_LST_SR"]
+                elif self.fwd_analysis:
+                    import_sr_cat_dict = select_cat_dict["FWD_CH_LST_SR"]
+                else:
+                    import_sr_cat_dict = select_cat_dict["TOP22_006_CH_LST_SR"]
 
-            # This dictionary keeps track of which selections go with which CR categories
-            import_cr_cat_dict = select_cat_dict["CH_LST_CR"]
+            if not self._skip_control_regions:
+                # If we are not skipping the control regions, we will import the CR categories
+                # This dictionary keeps track of which selections go with which CR categories
+                import_cr_cat_dict = select_cat_dict["CH_LST_CR"]
+                if self.tau_h_analysis:
+                    import_cr_cat_dict.update(select_cat_dict["TAU_CH_LST_CR"])
```

Comment:
Is the tau region the only one with extra CRs?

Reply:
yes, for now